lib/standard_deviation_module.rb

```diff
@@ -0,0 +1,29 @@
+module StandardDeviation
+
+  def standard_deviation_age
```

Comment:
This method should take the array of integers as an argument. This will only work if whatever is using this module has something called `ages`

Reply:
This doesn't make sense to me upon further review. Lets go over it on Tuesday if we have time.